website/static/js/pages/project-dashboard-page.js

```diff
@@ -93,6 +95,29 @@ var institutionLogos = {
     }
 };
 
+// Load categories to pass in to create project
+var loadCategories = function() {
+    var deferred = m.deferred();
+    var message = 'Error loading project category names.';
+    m.request({method : 'OPTIONS', url : $osf.apiV2Url('nodes/', { query : {}}), config : mHelpers.apiV2Config({withCredentials: window.contextVars.isOnRootDomain})})
+        .then(function _success(results){
+            if(results.actions && results.actions.POST.category){
+                var categoryList = results.actions.POST.category.choices;
+                categoryList.sort(function(a, b){ // Quick alphabetical sorting
+                    if(a.display_name < b.display_name) return -1;
+                    if(a.display_name > b.display_name) return 1;
+                    return 0;
+                });
+                deferred.resolve(categoryList);
+            }
+            deferred.reject(message);
```

Comment:
Minor: It might be useful to distinguish this "error" from the one a few lines down. We should never get to this point, so logging with Raven would be useful as well.

```js
errorMsg = 'API returned as success response, but no categories were fetched.';
Raven.captureMessage(message);
deferred.reject(errorMsg);
```